# Goal
int32 goal_id
---
# Result
int32 result_value
string message
---
# Feedback
int32 progress
